cmd/rig-ops/cmd/plugins/setup.go

```diff
@@ -120,5 +120,21 @@ The dry run will be executed with the resulting list of plugins.`,
 	}
 	pluginsCmd.AddCommand(list)
 
+	computeConfig := &cobra.Command{
+		Use: "compute-config",
+		//nolint:lll
+		Short: "Given an operator config and a capsule spec, computes the configuration generated for each plugin.",
```

Comment:
Perhaps we should allow the config to be a values file as well?

Reply:
Yes that is on my todo list to enable both